vnext/Directory.Build.props

```diff
@@ -33,30 +33,26 @@
   -->
   <PropertyGroup>
     <ReactNativeWindowsDir Condition="'$(ReactNativeWindowsDir)' == ''">$(MSBuildThisFileDirectory)</ReactNativeWindowsDir>
-    <ReactNativePackageDir Condition="'$(ReactNativePackageDir)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'node_modules\react-native\package.json'))\node_modules\react-native\</ReactNativePackageDir>
-
-    <RootIntDir>$(ReactNativeWindowsDir)\build</RootIntDir>
```

Comment:
Thanks, Julio recommended getting these out, but I never around got to them. They stem from when we were running out of disk space on the hosted agents.